docker-compose.yml

```diff
@@ -50,18 +50,18 @@ services:
       REPMGR_NODE_NAME: pg-0
       REPMGR_NODE_NETWORK_NAME: pg-0
       REPMGR_PORT_NUMBER: 5432
-  # xlm_r_bert_base_nli_stsb_mean_tokens:
-  #   build: .
-  #   platform: linux/x86_64
-  #   command: ["make", "run_model"]
-  #   volumes:
-  #     - ".:/app"
-  #   depends_on:
-  #     - redis
-  #   env_file:
-  #     - .env_file
-  #   environment:
-  #     MODEL_NAME: meantokens
+#  xlm_r_bert_base_nli_stsb_mean_tokens:
+#   build: .
+#   platform: linux/x86_64
+#   command: ["make", "run_model"]
+#   volumes:
+#     - ".:/app"
+#   depends_on:
+#     - redis
+#   env_file:
+#     - .env_file
+#   environment:
+#     MODEL_NAME: meantokens
   # indian_sbert:
```

Comment:
resolved in the new commit

Reply:
This is not resolved. You've done the reverse of what was needed. Please add back the space in front of each `#` so that the lines do not show as changed in the diff

Reply:
I've removed the leading spaces to solve indention problem in docker-compose.yml 
Uncommenting that part of code and then running docker compose up lead to that error `yaml: line 6: did not find expected key` and to solve it, I needed to remove the leading spaces. So I applied the same to the rest of vector models code so they would work by uncommenting them without further corrections

Reply:
I've put them back. In VS Code you can use Ctrl + / to comment and uncomment a block. It will take care of adding # followed by a space or removing it the same.